public/components/trace_analytics/components/common/helper_functions.tsx

```diff
@@ -677,3 +675,83 @@ export const parseHits = (payloadData: string): ParsedHit[] => {
     return [];
   }
 };
+
+export const InjectElementsIntoGrid = (
+  rowCount: number,
+  maxDisplayRows: number,
+  tracesTableMode: string,
+  loadMoreHandler?: () => void
+) => {
+  useEffect(() => {
+    setTimeout(() => {
+      const toolbar = document.querySelector('.euiDataGrid__controls') as HTMLElement;
+      if (toolbar && rowCount > maxDisplayRows) {
+        toolbar.style.display = 'flex';
+        toolbar.style.alignItems = 'center';
+        toolbar.style.justifyContent = 'space-between';
+
+        let warningDiv = toolbar.querySelector('.trace-table-warning') as HTMLElement;
+        if (!warningDiv) {
+          warningDiv = document.createElement('div');
+          warningDiv.className = 'trace-table-warning';
+          warningDiv.style.marginLeft = 'auto';
+          warningDiv.style.color = '#a87900';
```

Comment:
is this visible in both dark and light mode? if not oui might have some class names for coloring

Reply:
<img width="224" alt="Screenshot 2025-03-28 at 12 56 24 PM" src="https://github.com/user-attachments/assets/964837d7-dc10-4dab-aa12-568c0b51fa4b" />
<img width="185" alt="Screenshot 2025-03-28 at 12 56 52 PM" src="https://github.com/user-attachments/assets/f9409597-4b08-466b-98e2-836808868ecb" />

Warning Color appears visible in both modes